### PR TITLE
New UI proposal - merge changes

### PR DESCRIPTION
**Pull request by lorenz10**

### CODE REVIEW

**Comment by lorenz10** [comment]
I think I have included all chenges made in the other pull request.
I mainly changed what's inside the build() methods, leaving almost untouched the logic of the app.
There are margin of improvement for address book and some screens has been left untouched for the moment.

Screenshot_1625178539
Screenshot_1625210371
Screenshot_1625228850
Screenshot_1625229376
Screenshot_1625229712
Screenshot_1625229977

**Comment by bananenwilly** [comment]
Screenshot_1625318364
still empty green screen when no transactions are available 

Screenshot_1625318442
I would reduce the space between the items - that gap seems a little big 

Screenshot_1625318528
purchase button has no caption

Screenshot_1625318585
top item needs a bit of space to breath

Screenshot_1625318641
Dark mode is important to a lot of people and your current color choices are pretty low contrast or unreadable.  

Screenshot_1625318756
Confirmed/Unconfirmed balance has no color distinction any more - they need to

Screenshot_1625318858
i feel like the in/all/out selection tiles could use a visual divider from the balance

**Comment by bananenwilly** [comment]
another thing: I'm aware that the current version does not look very pretty on tablets either. I think you've mentioned it in the old PR before, but those cards are too wide on iPads etc.

**Comment by lorenz10** [comment]
> still empty green screen when no transactions are available
> I would reduce the space between the items - that gap seems a little big
> purchase button has no caption
> top item needs a bit of space to breath

First four screens solved.

> Dark mode is important to a lot of people and your current color choices are pretty low contrast or unreadable.
> Confirmed/Unconfirmed balance has no color distinction any more - they need to

Dark mode when I'll have time, the colors are the ones I've found...

> i feel like the in/all/out selection tiles could use a visual divider from the balance

I don't know

> Another thing: I'm aware that the current release does not look very pretty on tablets... I think you've mentioned it in the old PR before, but those cards are too wide on iPads etc.

in portrait or landscape?

**Comment by bananenwilly** [comment]
Tablets: portrait
another thing is the gap between connected and the  icon

File: assets/translations/en.json [hunk 1]
```diff
@@ -155,6 +156,7 @@
     "time": "Time",
     "transaction_details": "Transaction details",
     "transactions_all": "All",
+    "transactions_empty": "Empty list",
```
Comment (bananenwilly) [inline assets/translations/en.json:159]: check transactions_none

**Comment by lorenz10** [comment]
Check dark mode draft

File: lib/screens/wallet_home.dart [hunk 2]
```diff
@@ -180,57 +187,83 @@ class _WalletHomeState extends State<WalletHomeScreen>
...
           ),
           BottomNavigationBarItem(
-            icon: Icon(Icons.list),
+            icon: Icon(Icons.list_rounded),
             label: AppLocalizations.instance.translate('wallet_bottom_nav_tx'),
+            backgroundColor: back,
+          ),
+          BottomNavigationBarItem(
+            icon: Icon(Icons.menu_book_rounded),
+            label: 'Addresses',
```
Comment (bananenwilly) [inline lib/screens/wallet_home.dart:246]: that one needs translation

File: lib/screens/wallet_home.dart [hunk 3]
```diff
@@ -272,49 +305,144 @@ class _WalletHomeState extends State<WalletHomeScreen>
...
+                  body,
+                ],
+              ),
             ),
     );
   }
 }
+
+
+/// Widgets that should be moved from here... **/
```
Comment (bananenwilly) [inline lib/screens/wallet_home.dart:321]: I would appreciate that, hehe
maybe into tools or widgets folder?

File: lib/widgets/wallet_home_connection.dart [hunk 4]
```diff
@@ -8,47 +8,44 @@ class WalletHomeConnection extends StatelessWidget {
...
+      children: [
+        Image.asset(
+          'assets/icon/ppc-icon-white-256.png',
+          width: 20,
+        ),
+        SizedBox(
+          width: 10,
+        ),
+        SizedBox(
+          width: MediaQuery.of(context).size.width / 4,
```
Comment (bananenwilly) [inline lib/widgets/wallet_home_connection.dart:45]: that one creates a pretty big gap on tablets

**Pull request merged**